### DISCUSSION TITLE
Is there available column summary option?

### DISCUSSION THREAD

**Discussion by karthickTT** [fiduswriter/Simple-DataTables#343] [General]
Hi,

Is there option available sum column values at the footer?

Thanks

**Comment by johanneswilm** [comment]
What would the HTML you would expect look like? Or maybe there are examples of other tables out there that do this?

**Comment by karthickTT** [comment]
Hi,
Please find the screen short
![datatable](https://github.com/fiduswriter/simple-datatables/assets/122593579/7c087a71-b25f-4b85-83f7-aaa68086be17)

https://qawall.in/wp-content/uploads/2022/04/sum-of-datatable-column-2-1160x551.png

This is what i am expecting..

**Comment by johanneswilm** [comment]
right, where is that from? I need to see the HTML structure.

**Reply by karthickTT** [reply]
did you get any idea with the reference?

**Comment by karthickTT** [comment]
The image took from online for just reference.
Check the below code for the HTML reference.

https://codepen.io/RedJokingInn/pen/PbrqWa

https://datatables.net/examples/advanced_init/footer_callback.html


Note: Initally sum all column values & while filter, sum the values which are filtered.

**Comment by johanneswilm** [comment]
Yes, I think it should be possible by using the renderTable method. But I haven't had the time to try it out. Please go ahead if you want to try.

**Comment by karthickTT** [comment]
I am also busy with some other things. I will update it, if i work on it.
Let me know, if you did anything.
thanks johanneswilm

**Comment by johanneswilm** [comment]
Thanks to @SandroHc  in version 9.0.0, there is now an option to add a table footer. See https://fiduswriter.github.io/simple-datatables/demos/24-footer/
